deployment/address_book.go

```diff
@@ -93,20 +93,22 @@ type AddressBookMap struct {
 	mtx              sync.RWMutex
 }
 
-// save will save an address for a given chain selector. It will error if there is a conflicting existing address.
-func (m *AddressBookMap) save(chainSelector uint64, address string, typeAndVersion TypeAndVersion) error {
-	_, exists := chainsel.ChainBySelector(chainSelector)
-	if !exists {
+// Save will save an address for a given chain selector. It will error if there is a conflicting existing address.
+func (m *AddressBookMap) Save(chainSelector uint64, address string, typeAndVersion TypeAndVersion) error {
```

Comment:
Let's update the Save test with some other family addresses

Reply:
why did this become public? i don't see anything that using it in other packages.

Reply:
case added!

Reply:
yeah +1 @krehermann this should remain a private helper, there's already a public Save. Not sure how this actually compiles it looks like it has the same signature as the public Save?